.github/workflows/examples.yml

```
@@ -26,7 +26,33 @@ jobs:
           if-no-files-found: error
           compression-level: 0
 
-  test-examples:
+  test-examples-params:
+    needs: build-release
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        example: [
+          "patrol-events",
```

```suggestion
          "events",
```
Rename for compatibility with #281 / #330